schemas/simple-bus.yaml

```diff
@@ -14,7 +14,7 @@ maintainers:
 
 properties:
   $nodename:
-    pattern: "^(bus|soc|axi|ahb|apb)(@[0-9a-f]+)?$"
+    pattern: "^([a-zA-Z][a-zA-Z0-9,+\\-._]+-bus|bus|soc|axi|ahb|apb)(@[0-9a-f]+)?$"
```

Comment:
This should be limited to '[a-zA-Z0-9\-]'.

Reply:
The names in question are "motherboard" and "iofpga" at the moment, so I needed a .dts patch to at least append "-bus" to that.
Labels don't help when the kernel talks about a DT path as "bus/bus/bus/device", so I would rather allow this to be more descriptive.